api/lib/models/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var Sequelize = require('sequelize');
+var logger = require('../log/logger');
+var sequelizeURL = process.env.DB_URI;
+var sequelize = new Sequelize(sequelizeURL);
```

Comment:
Sequelize uses a default connection pool which we don't override as of now. We can look into this if required.